airbyte-integrations/connectors/source-sftp-bulk/integration_tests/integration_test.py

```diff
@@ -21,14 +23,17 @@ def test_check_invalid_private_key_config(configured_catalog: ConfiguredAirbyteC
             "private_key": "-----BEGIN OPENSSH PRIVATE KEY-----\nbaddata\n-----END OPENSSH PRIVATE KEY-----",
         }
     }
-    outcome = SourceSFTPBulk(catalog=configured_catalog, config=invalid_config, state=None).check(logger, invalid_config)
-    assert outcome.status == Status.FAILED
+    with pytest.raises(AirbyteTracedException) as exc_info:
+        SourceSFTPBulk(catalog=configured_catalog, config=invalid_config, state=None).check(logger, invalid_config)
+
+    assert exc_info.value.failure_type.value == FailureType.config_error.value
 
 
 def test_check_invalid_config(configured_catalog: ConfiguredAirbyteCatalog, config: Mapping[str, Any]):
     invalid_config = config | {"credentials": {"auth_type": "password", "password": "wrongpass"}}
-    outcome = SourceSFTPBulk(catalog=configured_catalog, config=invalid_config, state=None).check(logger, invalid_config)
-    assert outcome.status == Status.FAILED
+    with pytest.raises(AirbyteTracedException) as exc_info:
+        SourceSFTPBulk(catalog=configured_catalog, config=invalid_config, state=None).check(logger, invalid_config)
+    assert exc_info.value.failure_type.value == FailureType.config_error.value
```

Comment:
airbyte exceptions are now [caught here](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/entrypoint.py#L154-L164), so it is ok to change the test a little bit as we don't call the entry point but directly the check method in the source.